README.md

```diff
@@ -1,13 +1,25 @@
 # tigerbeetle-node
-[TigerBeetle](https://github.com/coilhq/tigerbeetle) client for NodeJS
+[TigerBeetle](https://github.com/coilhq/tigerbeetle) client for NodeJS.
 
 ## Installation
-**Prerequisites:** Node >= 14.0.0. Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
+The follow steps will install the `tigerbeetle-node` Node library to your local.
+
+### Prerequisites 
+
+* NodeJS >= `14.0.0`. _(If the correct version is not installed, an installation error will occur)_ 
+
+> Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
+
+> Ensure you are not currently in the project (tigerbeetle-node) directory, but the `parent` directory instead;
```

Comment:
Did you run into this @koekiebox ? I think we can skip this as a prerequisite, since we do already explain that the module will be installed to the current working directory, and this is how these particular package managers work.

Reply:
@jorangreef  I tried installing via `npm install .`, but the installer assumes you are in the parent directory due to postinstall "see below". 

# Error `npm`:
```
npm ERR! command sh -c ./scripts/postinstall.sh
```

# Error `yarn`:
```
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 0s 605ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ @types/node@npm:14.17.20 can't be found in the cache and will be fetched from the remote registry
```